Daniel0897/lesson6/homework6.rb

```diff
@@ -0,0 +1,96 @@
+a =  [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+#Use the each method of Array to iterate over [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], and print out each value.
+a.each { |item| puts item }
+#Same as above, but only print out values greater than 5.
+a.each { |item| puts item if item >= 5 }
+
+#Now, using the same array from 2, use the select method to extract all odd numbers into a new array.
+b = a.select.each_with_index { |_, i| i.odd? }
```

Comment:
It think this will be much simpler
```suggestion
a.select { |number| number.odd? }
```
or this
```suggestion
a.select(&:odd?)
```